config/config.go

```diff
@@ -24,7 +25,10 @@ func LoadFile(filename string) (*Config, error) {
 
 var (
 	DefaultModule = Module{
-		Version: 2,
+		Version:        2,
+		Retries:        3,
+		MaxRepititions: 25,
+		Timeout:        60,
```

Comment:
> I would suggest 20 seconds as the current hard coded values called for 3 retries in 60 seconds.

The timeout is per query, not overall. It'd be a very unusual network that had more than 1s latency locally.

> I had initially attempted to use int but found that Go did not want to do automatic type casting since it was not a constant (60 is a constant so I assume that the compiler automatically does the type cast at compilation time). 

Try `time.Second` as the default.